Sources/MapFilter/StepSlider.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+protocol StepSliderDelegate: AnyObject {
+    func stepSlider<StepValueKind>(_ stepSlider: StepSlider<StepValueKind>, didChangeValue value: Float)
+    func stepSlider<StepValueKind>(_ stepSlider: StepSlider<StepValueKind>, didChangeRoundedStepValue value: StepValueKind)
+}
+
+class StepSlider<StepValueKind: Comparable & SliderReferenceValue>: UISlider {
```

Comment:
I would probably put the UISlider directly in the ValueSlider class. Should probably make a slider component together with the range filter thingy

Reply:
I'm not sure it would be possible to handle `accessibilityDecrement` and `accessibilityIncrement` then though. 

The ValueSliderControl would need to be made into something that can be both range and single slider then. Not sure how complex it would make that class. Perhaps better to look into that when this slider is used for ranges too?

Reply:
Jeje, not in this PR ;)